addons/beehave/nodes/beehave_node.gd

```diff
@@ -27,3 +28,14 @@ func tick(actor: Node, blackboard: Blackboard) -> int:
 ## Called when this node needs to be interrupted before it can return FAILURE or SUCCESS.
 func interrupt(actor: Node, blackboard: Blackboard) -> void:
 	pass
+
+
+## Called before the first time it ticks by the parent.
+func enter(actor: Node, blackboard: Blackboard) -> void:
+	pass
+
+
+## Called after the last time it ticks and returns 
+## [code]SUCCESS[/code] or [code]FAILURE[/code].
```

Comment:
Maybe worth mentioning that this happens after it previously returned `RUNNING`

Also, what would happen on nodes that never return `RUNNING` here - would they call `enter()` and `exit()` every tick?